autofaiss/external/build.py

```diff
@@ -233,11 +235,13 @@ def create_index(
                 batch_size=batch_size,
                 embedding_ids_df_handler=embedding_ids_df_handler,
                 temporary_indices_folder=temporary_indices_folder,
+                nb_indices_to_keep=nb_indices_to_keep,
             )
         else:
             raise ValueError(f'Distributed by {distributed} is not supported, only "pyspark" is supported')
-    # Give standard values for index hyperparameters if possible.
-    if any(re.findall(r"OPQ\d+_\d+,IVF\d+_HNSW\d+,PQ\d+", index_key)):
-        set_search_hyperparameters(index, f"nprobe={64},efSearch={128},ht={2048}", use_gpu)
+    if nb_indices_to_keep > 1:
```

Comment:
if there is only one index we don't set the hp params ?

Reply:
This is a mistake, I wanted to type `if nb_indices_to_keep == 1:` actually. Will fix that.